internal/nostr/nostr.go

```diff
@@ -713,7 +713,6 @@ func (svc *Service) startSubscription(ctx context.Context, subscription *Subscri
 	var relay *nostr.Relay
 	var isCustomRelay bool
 	var err error
-	waitToReconnectSeconds := 0
```

Comment:
A few lines down (line 726) we cancel the subscription and return if there is an error. I think we should only exit this loop when the `context.Done` channel receives a message, and we should handle it in one place like we do in Alby Hub.